console/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+---
+services:
+
+  caddy:
+    image: lucaslorentz/caddy-docker-proxy:ci-alpine
+    container_name: caddy
+    ports:
+      - 80:80
+      - 443:443
+    environment:
+      - CADDY_INGRESS_NETWORKS=caddy
+    networks:
+      - caddy
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - caddy_data:/data
+    restart: unless-stopped
+    depends_on:
+      - autobase-console-ui
+      - autobase-console-api
```

Comment:
Those are not needed and may be inefficient as they will start containers in a sequence. `depends on` container is used mostly when the start of a service requires another service to get started and work, like the API requires the DB to start, without the DB, the API will shut down. Frontend and Caddy do not require other services to be started before, the container will be working even if the other services are not yet running. Caddy and UI may not respond with an OK response, but this is an application-level responsibility rather than a container one.